CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version `v0.2.4` is auto-generated.
 
+## [v5.0.3](https://github.com/puppetlabs/puppetserver-helm-chart/tree/v5.0.3) (2020-11-01)
+
+- Fix for r10k_code_cronjob.sh and r10k_hiera_cronjob.sh syntax error with `map[]`
+
+[Full Changelog](https://github.com/puppetlabs/puppetserver-helm-chart/compare/v5.0.1...v5.0.3)
```

Comment:
The `v5.0.1` needs to be changed to `v5.0.2`.

Reply:
OK. Will fix it and submit again.